person.rb

```diff
@@ -4,20 +4,26 @@
 
 class Person < Nameable
   attr_accessor :name, :age
-  attr_reader :id
+  attr_reader :id, :rentals
 
   def initialize(age, name: 'unknown', parent_permission: true)
     super()
     @id = Random.rand(1..1000)
     @name = name
     @age = age
     @parent_permission = parent_permission
+    @rentals = []
   end
 
   def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals << rental
+    rental.person = self
+  end
```

Comment:
- [ ] like the previous reviewer noted, The person itself should be sent as a parameter to create the new rental achieving the 'has-many' association

```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
```